tests/src/Unit/EventSubscriber/CacheInvalidatorSubscriberTest.php

```diff
@@ -92,7 +101,9 @@ public function testInvalidCacheTags() : void {
   public function testValidCacheTags() : void {
     $mock = new CacheTagInvalidator();
     $environmentResolver = $this->getEnvironmentResolver();
-    $sut = new CacheTagInvalidatorSubscriber($mock, $environmentResolver);
+    $eventDispatcher = $this->prophesize(EventDispatcherInterface::class);
+    $eventDispatcher->dispatch(Argument::any(), Argument::any())->willReturn(new Event());
```

Comment:
We should use the new `CacheTagInvalidateEvent` class here and make sure the event is actually dispatched:
```php
    $eventDispatcher->dispatch(...)
      ->shouldBeCalled();
```